tools/table_annovar/table_annovar.xml

```diff
@@ -42,7 +42,7 @@
         #set operation = ','.join( $operation )
 
         ## Annotate variants.
-        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
+        \${SCRIPT_PATH}/table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
```

Comment:
and that's were I went wrong...
You can go ahead and close this pull request if you like.

Thanks for clearing this up.